x-pack/plugins/ml/server/models/job_validation/messages.json

```diff
@@ -151,6 +151,10 @@
     "status": "ERROR",
     "text": "The model memory limit is greater than the max model memory limit configured for this cluster."
   },
+  "mml_value_invalid": {
+    "status": "ERROR",
+    "text": "{{mml}} is not a valid value for model memory limit. e.g. 10MB"
```

Comment:
I wonder if something like `{{mml}} is not a valid value for model memory limit. The value should be specified in bytes e.g. 10MB` might be clearer for the error text?

Reply:
Thanks for the suggestion, I updated the message in 70311b662f.